src/main.cpp

```diff
@@ -12,25 +12,25 @@ int main(int argc, char *argv[]) {
 	std::string filename;
 	driver drv;
 	if (argv[argc - 1] == std::string("-h")) {
-		Frontend::Helper(helpArgument);
-		return 0;
+		Frontend::PrintHelper();
+		return 1;
 	}
-
 	for (int i = 1; i < argc; ++i) {
 		if (file_included(argv[i])) {
 			filename = argv[i];
 		} else if (argv[i] == std::string("-o")) {
 			frontend.outputFileName = std::string(argv[++i]);
+		} else {
+			Frontend::Exception(argError, argv[i]);
+			return 1;
 		}
 	}
-
 	if (drv.parse_file(filename) == 0) {
 		frontend.drv = &drv;
 		frontend.WriteFile();
```

Comment:
It should probably not end up returning 1 when we successfully parse the file.

Reply:
What do you mean?